src/frontend/src/pages/EquipmentBrowser/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Switch, Route, Link, useRouteMatch } from 'react-router-dom';
+import Button from '../../components/Button';
+import Header from '../../components/Header';
```

Comment:
This is not the header #33 is talking about. The header component needs to be created according to the design mock in Figma.